src/main/resources/db/changelog/changes/20240617-bv.yaml

```diff
@@ -0,0 +1,84 @@
+databaseChangeLog:
+  - changeSet:
+      id: 2024-06-17-1
+      author: Victor
+      changes:
+        - createTable:
+            tableName: users
+            columns:
+              - column:
+                  name: id
+                  type: BIGINT
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: email
+                  type: VARCHAR(255)
+                  constraints:
+                    nullable: false
+                    unique: true
+              - column:
+                  name: date_of_birth
+                  type: TIMESTAMP
+                  constraints:
+                    nullable: false
+
+  - changeSet:
+      id: 2024-06-17-2
+      author: Victor
+      changes:
+        - insert:
+            tableName: users
+            columns:
+              - column:
+                  name: email
+                  value: "beca@yahoo.com"
+              - column:
+                  name: date_of_birth
+                  value: "1995-02-20T00:00:00"
+
+  - changeSet:
+      id: 2024-06-17-3
+      author: Victor
+      changes:
+        - insert:
+            tableName: users
+            columns:
+              - column:
+                  name: email
+                  value: "vlad@bossu.com"
+              - column:
+                  name: date_of_birth
+                  value: "1980-05-05T00:00:00"
+
+  - changeSet:
+      id: 2024-06-17-4
+      author: Victor
+      changes:
+        - createTable:
+            tableName: orders
+            columns:
+              - column:
+                  name: id
+                  type: BIGINT
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: delivery_date
+                  type: TIMESTAMP
+                  constraints:
+                    nullable: false
+              - column:
+                  name: order_date
+                  type: TIMESTAMP
+                  constraints:
+                    nullable: false
+              - column:
+                  name: user_id
+                  type: BIGINT
+                  constraints:
+                    nullable: false
```

Comment:
tu nullable asta de ce il pui?

Reply:
sa nu fiu null id ul, sa am o valoare pentru el, dar nu stiu daca e nevoie de el aici avand in vedere ca se completeaza automat